handlers/map-tactile-svg/map-svg.py

```diff
@@ -263,6 +274,10 @@ def handle():
             logging.debug("Expected to obtain string as "
                           "POI name in nominatim ")
             logging.debug("Obtained type " + str(type(targetTag)))
+        except ValueError:
+            logging.debug("Expected to obtain string as "
```

Comment:
Suggest logging the message included with the error and not this hard-coded one. It might make future maintenance easier. (Some for above `except` to the extent that is possible.)